pkg/monitor/intervalcreation/node.go

```diff
@@ -46,6 +55,15 @@ func IntervalsFromEvents_NodeChanges(events monitorapi.Intervals, beginning, end
 		}
 		reason := strings.SplitN(strings.TrimPrefix(event.Message, "reason/"), " ", 2)[0]
 
+		var roles string
```

Comment:
factor into a function I can call kind of like https://github.com/deads2k/origin/blob/bbd9949c834a1491ac62a9b74ea3fdb4bb5f6840/pkg/monitor/monitorapi/identification_operator.go#L10-L11

Reply:
updated. see 188c0fcfd11f19281d8c86d3bfeba0c27042754a.